src/main/groovy/jenkins/automation/utils/BddSecurityJobBuilder.groovy

```diff
@@ -0,0 +1,47 @@
+package jenkins.automation.utils
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+
+
+class BddSecurityJobBuilder {
+
+    String name
+    String description
+    String baseUrl
+
+    Job build(DslFactory factory){
+        factory.job(name){
+            it.description this.description
+
+            steps {
+                shell("""umask 002
+                        #export http_proxy=localhost:8888
```

Comment:
The commented line are the configuration for the job it will produce, not code here per se. When porting the jobs, it can be included for a little while.